### ISSUE TITLE
MySQL Group permissions on log folder

### ISSUE THREAD

**Issue by tomjn** [Varying-Vagrant-Vagrants/VVV#1557]
It would appear that the mysql user/group is unable to write to `/var/log/mysql` because they don't have permission to write to the folder. This is because it is owned by `www-data` and a synced vagrant folder, so any files in the `/var/log` folder will be owned by `www-data`.

This causes an issue on package installation that is blocking, #1553 and #1556 and might be a cause of #1504

**Comment by tomjn** [comment]
Found by running:

```shell
vagrant@vvv:~$ sudo mysql_install_db
Installing MariaDB/MySQL system tables in '/var/lib/mysql' ...
2018-06-19 13:59:57 0 [ERROR] mysqld: File '/var/log/mysql/mariadb-bin.index' not found (Errcode: 13 "Permission denied")
2018-06-19 13:59:57 0 [ERROR] Aborting
```

**Comment by tomjn** [comment]
fixed by #1553
